explorer/testdata/class/base_constructor.carbon

```diff
@@ -0,0 +1,34 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// RUN: %{explorer-run}
+// RUN: %{explorer-run-trace}
+// CHECK:STDOUT: 1
+// CHECK:STDOUT: 2
+// CHECK:STDOUT: result: 0
+
+package ExplorerTest api;
+
+base class A {
+  fn Create() -> Self {
+    return {.value_a = 1};
+  }
+  var value_a: i32;
+}
+
+class B extends A {
+  fn Create() -> Self {
+    // Note: This syntax is supported per the design, but currently XFAIL for the explorer.
```

Comment:
Looks like this is working, is the comment out of date?

Reply:
Yes thank you, done.